internal/analytics/analytics.go

```diff
@@ -127,7 +128,11 @@ func (d *customDimensions) toMap() map[string]string {
 		"9":  d.machineID,
 		"10": d.projectName,
 		"11": d.sessionToken,
+		"12": d.uniqID,
 	}
+	logging.Debug("Custom dimensions analytics map formed as: %v", m)
```

Comment:
I was suggesting this for you to test things locally. Actually logging this is going to be too verbose as it would happen for every single analytics event.